src/libraries/System.Reflection.Emit/src/System/Reflection/Emit/TypeBuilderImpl.cs

```diff
@@ -148,15 +140,12 @@ private void ValidateMethods()
                     }
 
                     // If this is an abstract method or an interface should not have body.
+                    ILGeneratorImpl? body = method.ILGeneratorImpl;
                     if (body != null && body.ILOffset > 0)
                     {
                         throw new InvalidOperationException(SR.Format(SR.InvalidOperation_BadMethodBody, method.Name));
                     }
                 }
-                else if ((body == null || body.ILOffset == 0) && !method._canBeRuntimeImpl)
-                {
-                    throw new InvalidOperationException(SR.Format(SR.InvalidOperation_BadEmptyMethodBody, method.Name));
```

Comment:
This is the only use of InvalidOperation_BadEmptyMethodBody string in System.Reflection.Emit. It can be deleted.